src/components/AboutPage/index.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import flowchartImg from "../../assets/overview-image.png";
+import agentRepOptionsImg from "../../assets/agent-rep-options.png";
+import VisualGlossary from "../VisualGlossary";
+import ContentPage from "../ContentPage";
+import ContentPagePanel from "../ContentPagePanel";
+
+import styles from "./style.css";
+
+const AboutPage = (): JSX.Element => {
+    return (
+        <ContentPage>
+            <ContentPagePanel>
+                <h1>About Simularium</h1>
+                <p className={styles.intro}>
+                    With this first release we present the Simularium Viewer, an
+                    online visual analysis tool, which is one component of the
+                    larger Simularium platform. The Simularium Viewer provides
+                    an online application for visualizing simulation
+                    trajectories and related plots.
+                </p>
+            </ContentPagePanel>
+
+            <ContentPagePanel isDark={true} isWide={true}>
+                <h2>Visual Guide</h2>
+                <VisualGlossary />
+            </ContentPagePanel>
+
+            <ContentPagePanel>
+                <h2>Simularium v1.0</h2>
+                <div className={styles.leftAlign}>
+                    <p>
+                        Simularium v1.0 allows you to interact with
+                        precalculated simulation results — including your own
+                        results converted into the Simularium file format
+                        (JSON). The Simularium Viewer uses advanced rendering
+                        techniques developed by Le Muzic et al. (2015) and
+                        Mindek et al. (2017) to enable meaningful interpretation
+                        of spatial relationships among thousands of moving
+                        components. Users can import a trajectory file (JSON
+                        format) from their computer, or they can stream provided
+                        examples by choosing one from the Load model dropdown or
+                        by clicking on a card above. Future versions of
+                        Simularium will allow you to modify, run, and even
+                        create simulations.
+                    </p>
+                    <h3>Current object representtion options:</h3>
+                    <p>
+                        Spheres: by default, each agent in a scene is
+                        represented as a single sphere Mesh surfaces: represent
```

Comment:
Looks like these should have line breaks between them? One each for for "Spheres", "Mesh surfaces", etc.